kong/db/strategies/off/connector.lua

```diff
@@ -0,0 +1,82 @@
+local meta = require "kong.meta"
+
+
+local OffConnector   = {}
+OffConnector.__index = OffConnector
+
+
+local function ignore()
+  return true
+end
+
+
+function OffConnector.new(kong_config)
+  local self = {
+    timeout = 1,
+    close = ignore,
+    connect = ignore,
+    truncate_table = ignore,
+    truncate = ignore,
+  }
+
+  return setmetatable(self, OffConnector)
+end
+
+
+function OffConnector:infos()
+  return {
+    strategy = "off",
+    db_name = "kong_off",
+    db_desc = "cache",
+    db_ver = meta._VERSION,
+  }
+end
+
+
+function OffConnector:connect_migrations(opts)
+  -- FIXME what is this used for, again?
+  return {}
+end
+
+
+function OffConnector:query()
+  return nil, "cannot perform queries without a database"
+end
+
+
+do
+  -- migrations
+
```

Comment:
What is the purpose of this `do` block?
Also, the whitespace here seems weird to me, though it might be to style guide.

Reply:
it was just to mirror the structure of `kong.db.strategies.cassandra.connector`.

> Also, the whitespace here seems weird to me, though it might be to style guide.

Agreed and yes.